<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BreakOut</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MainMenu" name="mainMenu" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="RecordsTable" name="recordsTable" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="ChoosingLevel" name="choosingLevel" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="Game" name="gameView" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <zorder>recordsTable</zorder>
   <zorder>mainMenu</zorder>
   <zorder>gameView</zorder>
   <zorder>choosingLevel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChoosingLevel</class>
   <extends>QWidget</extends>
   <header>interface/choosinglevel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainMenu</class>
   <extends>QWidget</extends>
   <header>interface/mainmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecordsTable</class>
   <extends>QWidget</extends>
   <header>interface/recordstable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Game</class>
   <extends>QWidget</extends>
   <header>interface/game.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
